app/lib/features/tasks/actions/create_task.dart

```diff
@@ -0,0 +1,433 @@
+import 'package:acter/common/actions/select_space.dart';
+import 'package:acter/common/drag_handle_widget.dart';
+import 'package:acter/common/providers/space_providers.dart';
+import 'package:acter/common/toolkit/buttons/inline_text_button.dart';
+import 'package:acter/common/extensions/options.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/common/widgets/spaces/select_space_form_field.dart';
+import 'package:acter/features/tasks/actions/select_tasklist.dart';
+import 'package:acter/features/tasks/providers/tasklists_providers.dart';
+import 'package:acter/features/tasks/widgets/due_picker.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:dart_date/dart_date.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger('a3::tasks::create_update_task_item');
+
+Future<void> showCreateTaskBottomSheet(
```

Comment:
This is just 'copied-over from somewhere' stuff I assume? Nothing changed in actual ?